test/entities/swaps/v2/swapV2.integration.test.ts

```diff
@@ -44,7 +42,7 @@ const WETH = TOKENS[chainId].WETH;
 const vault = VAULT[chainId];
 
 describe('SwapV2', () => {
-    let client: Client & PublicActions & TestActions & WalletActions;
+    let client: ViemClient & TestActions;
```

Comment:
I guess my lean would be to go with a type name of  `PublicWalletClient` for situations like this

As you mentioned, we are only using Viem so i dont think having it as part of type name adds much meaning